package.json

```diff
@@ -217,14 +216,9 @@
     "webpack": "4.44.2",
     "webpack-merge": "^5.7.3"
   },
-  "overrides": {
-    "storybook-addon-react-router-v6": {
-      "react": "17.0.1",
-      "react-dom": "17.0.1"
-    }
-  },
   "resolutions": {
-    "@graphql-codegen/cli/**/ts-node": "10.8.2"
+    "@graphql-codegen/cli/**/ts-node": "10.8.2",
```

Comment:
Might be worth checking if this resolution could be removed, as the original reason it was installed was due to Storybook! ([comment](https://github.com/evergreen-ci/spruce/pull/1368#issuecomment-1182025643))